### ISSUE TITLE
Make sure some namelist parameters have a default value

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#1062]
There's a couple namelist parameters we need to make sure are on the lnd_in file by default for the PPE work.

freelivfix_slope_wet
freelivfix_intercept

The list of all parameters for this work are here:

https://docs.google.com/spreadsheets/d/1OtkaO_uAmafWKR9kgtRC2Ge6d6fkhymngSpben5SJ_Q

There are still some hardcoded soil parameters on the list, but it looks like they aren't going to be dealt with.

**Comment by ekluzek** [comment]
@djk2120 and @katiedagon make sure I have this list correct.

**Comment by ekluzek** [comment]
Looks like these are the default values for these two in the code:

biogeochem/CNNDynamicsMod.F90:    freelivfix_intercept = 0.0117_r8
biogeochem/CNNDynamicsMod.F90:    freelivfix_slope_wET = 0.0006_r8

**Comment by ekluzek** [comment]
Note, there are several other namelist values that aren't by default in the namelist (such as deepmixing_mixfact, denitrif_nitrateconc_* and a few others). We could just make sure all of these are set to the default value, so they could be easily changed in a simulation.

**Comment by djk2120** [comment]
Agreed on freelivfix_intercept and freelivfix_slope_wET. Those were the only two params that popped up when I was trying to fetch defaults for params on the spreadsheet. I'll double-check with the group on whether there are any other namelist variables that we may be missing.

**Comment by katiedagon** [comment]
Looking at [this code](https://github.com/djk2120/CLM5PPE/blob/master/pyth/namelist_params.ipynb), do we also need to set the default value of cmb_cmplt_fact?

Also, according to the PPE google spreadsheet, denitrif_nitrateconc_* are in the netcdf params file.

**Comment by ekluzek** [comment]
Yes, looks like you are correct about denitrif_nitrateconc_coefficient,denitrif_nitrateconc_exponent being on the params file. And yes as part of #1053 I'll set defaults for cmb_cmplt_fact as well as separating it into two variables.

**Comment by ekluzek** [comment]
OK, it looks like denitrif_nitrateconc_* is on the params file, but ALSO on the namelist. The one actually used is on the paramsfile, so we should remove the namelist version.

**Comment by ekluzek** [comment]
It would be good to review and make sure all the namelist values that are real are set in the namelist (if needed).

**Comment by ekluzek** [comment]
Another good check would be to make sure all of the variables listed as "N" in the google doc are on the default namelist and set to something. Just as another check that we are getting all the namelist parameters handled.

**Comment by katiedagon** [comment]
I took a look at all the PPE namelist parameters (listed "N" in the google sheet) and they are all in the new namelist_defaults_ctsm.xml file that is part of your pull request, so we should be good to go with being able to assign and pull default values for all the namelist parameters in the PPE. Thanks @ekluzek!

One minor inconsistency I noticed is that for jmaxb1, the namelist defaults file has the default value of 0.17, whereas the actual value used (based on a previous simulation's lnd_in file) is 0.093563, so it must be getting overridden somewhere.

**Comment by ekluzek** [comment]
@katiedagon thanks for checking this for us to make sure we have everything here. That's always helpful to have many people looking over things.

I looked into jmaxb1 and it's currently 0.17, but was only recently changed from the previous value in June. So an older simulation (or one with an older version of CTSM) would have the value you list. I did check to make sure the right value is in the current namelist and it is. So as far as I see jmaxb1 is fine as it is right now. But, certainly let us know if you see anything else with it or anything else needed for the PPE work.
